rca/project_styleguide/templates/patterns/pages/enquire_to_study/enquire_form_page.html

```diff
@@ -44,7 +44,7 @@ <h1 class="title-area__heading heading heading--display-two" id="main-content">
                     {% if settings.enquire_to_study.EnquiryFormKeyDetails.content %}
                         <div class="layout__start-one layout__span-two layout__@large-span-one">
                             <div class="key-details">
-                                <div class="key-details__header">
+                                <div class="key-details__header enquire-form__key-details">
```

Comment:
Hey @patrickcuagan I wonder if these classes would be better suited to modifiers of `key-details__header`, rather than a hybrid?

Same for the `enquire-form__section` - would it make sense to apply these to modifiers of `form__content` or `section` instead?

I'm not close to this project, though, so if you think it'd be better to create a new class block then fair enough!

Reply:
Ah I see - I've added modifiers now. Naming them have been a bit trivial though so I've just thought of the best possible.